kythe/cxx/verifier/testdata/singleton_inspect_expected_error_fs.txt

```diff
@@ -0,0 +1,3 @@
+Could not verify all goals. The furthest we reached was:
+  singleton_inspect_input_fs.txt:1:5-1:34 Nonsingleton ref Nonsingleton
+  Goal: fact(<null>, /kythe/edge/ref, <null>, /, "")
```

Comment:
Is it possible (and worth it) to add a test for the message during error recovery ?